cmd/kubeadm/app/constants/constants.go

```diff
@@ -157,9 +157,9 @@ const (
 	ControllerManagerUser = "system:kube-controller-manager"
 	// SchedulerUser defines the well-known user the scheduler should be authenticated as
 	SchedulerUser = "system:kube-scheduler"
-	// MastersGroup defines the well-known group for the apiservers. This group is also superuser by default
+	// ControlPlaneGroup defines the well-known group for the apiservers. This group is also superuser by default
 	// (i.e. bound to the cluster-admin ClusterRole)
-	MastersGroup = "system:masters"
+	ControlPlaneGroup = "system:masters"
```

Comment:
please keep `MastersGroup` and `LabelNodeRoleMaster` because they contain "master" as a value.
[1]

Reply:
actually....let's see what others think and if we can change these too.

Reply:
This must be changed to `SystemPrivilegedGroup`